styleguide/STYLE.md

```diff
@@ -0,0 +1,174 @@
+Same, but written by [Microsoft](https://docs.microsoft.com/en-us/style-guide/grammar/nouns-pronouns). 
+
+## Style and formatting 
+This section covers the matters that go beyond language and provides guidelines for consistency and a unified look.  
+
+### Consistency
+Keep your writing consistent with itself and other docs. This means abbreviations, capitalization, hyphens, names of UI elements, etc.   
+
+### Title capitalisation  
+
+Use Title case in page headings:
+*"The Quick Brown Fox Jumps Over the Lazy Dog."*  
+Also known as headline style and capital case. All words capitalized, except for certain subsets defined by rules that are not universally standardized, often minor words such as "the" (as above), "of", or "and". The standardization is only at the level of house styles and individual style manuals. (See Headings and publication titles.) A simplified variant is start case, where all words, including articles, prepositions, and conjunctions, start with a capital letter.
+
+Use Sentence case in section headings: 
+*"The quick brown fox jumps over the lazy dog."*  
+The standard case used in English prose. Generally equivalent to the baseline universal standard of formal English orthography mentioned above; that is, only the first word is capitalized, except for proper nouns and other words which are generally capitalized by a more specific rule.
+
+Same, but [on Wikipedia](https://en.wikipedia.org/wiki/Capitalization#By_name_of_style).
+
+### Capital letters in proper names 
+Only capitalize the name of the product - e.g. 'GitHub organization' and not 'GitHub Organization'.  
+
+### Capital letters in UX elements
+Use title capitalization for names of tabs and buttons in the Buildkite interface - e.g. 'Personal Settings', 'Repository Providers', 'Save Organization Settings'.
+
+### Bullet lists  
+
+Capitalize the first word; no full stops at the end if it’s only one sentence. If there are two or more sentences in the list element, the final one will have a full stop.  
+
+Only use bullet lists in the body of the text and never - in page or section headings.
+
+
+See also what Microsoft has to say on [lists](https://docs.microsoft.com/en-us/style-guide/scannable-content/lists). 
+
+### Writing numbers  
+Write out numbers up to 10, then use digits - '58 bugs in this script and just two hours to fix them all!'
+Long numbers use commas to separate thousands - '100,000,000.00'. When in doubt, look [here](https://docs.microsoft.com/en-us/style-guide/numbers).  
+
+Do not use numbers in page headings. Only use numbers less than 10 in section headings.  
+No restrictions on using numbers in the body of the text.  
+
+### Time and date  
+Use 24hr time with hours and minutes, but not seconds. Include timezone. e.g. 17:00 AEST 
+More in [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/date-time-terms).
+
+### Non-breaking characters
```

Comment:
Do you think we should remove this section? Or better to leave it in?

Reply:
Killed the beast.